includes/wikia/services/WikiService.class.php

```diff
@@ -47,66 +52,73 @@ class WikiService extends WikiaModel {
 	 * @return array of $userIds
 	 */
 	public function getWikiAdminIds( $wikiId = 0, $useMaster = false, $excludeBots = false, $limit = null, $includeFounder = true ) {
-		wfProfileIn( __METHOD__ );
+		if ( !empty( $this->wg->FounderEmailsDebugUserId ) ) {
+			return [ $this->wg->FounderEmailsDebugUserId ];
+		}
 
-		$userIds = array();
-		if ( empty($this->wg->FounderEmailsDebugUserId) ) {
-			// get founder
-			$wikiId = ( empty($wikiId) ) ? $this->wg->CityId : $wikiId ;
-			$wiki = WikiFactory::getWikiById($wikiId);
-			if ( !empty($wiki) && $wiki->city_public == 1 ) {
-				if ($includeFounder) {
-					$userIds[] = $wiki->city_founding_user;
-				}
+		if ( !empty( $this->wg->EnableAnswers ) ) {
+			return [];
```

Comment:
Based on the old code it looks like it would be possible to return `$wiki->city_founding_user;` even when `!empty( $this->wg->EnableAnswers )`, is that a case we should support still?